lib/cryptography.js

```diff
@@ -28,7 +29,12 @@ module.exports.decrypt = (options) => {
     const decipher = crypto.createDecipheriv(decryptionAlgo, key, iv)
     let decrypted = decipher.update(ciphertext, 'hex', 'utf8')
     decrypted += decipher.final('utf8')
-    return decrypted
+    if (fs.existsSync(outputFilePath)) {
```

Comment:
Let's not create a backup file in case the file exist instead throw an error and return.